packages/sdks/javascript/src/api/generated/apis/users-passwords-api.ts

```diff
@@ -58,4 +59,22 @@ export class UsersPasswordsApiService extends ApiBaseService {
         const response = await this.post <void, PasswordUserSendResetLink>(requestUrl, passwordUserSendResetLink);
         return new ApiResponse(response);
     }
+
+    /**
+     * 
+     * @summary Set or update the current user\'s password
+     * @param {UserPasswordUpdate} userPasswordUpdate A JSON object containing the password data
+     */
+    public async updateUserMePassword(userPasswordUpdate: UserPasswordUpdate): Promise<ApiResponse<void>> {
```

Comment:
Hi @dax-side thanks for your contribution!

As you can see [in the header of this file](https://github.com/devopness/devopness/pull/2161/files#diff-8e7bb142d066f685474c7c3b885c80cd3bed22de6f74c2c4c030ac6dd30b6366L9) this is an auto-generated file and this, and no other file under the `**/generated` folders should be manually edited:

```ts
/**
 * ...
 * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
 * https://openapi-generator.tech
 * Do not edit the class manually.
 */
```